grails-app/controllers/au/org/ala/biocache/hubs/OccurrenceController.groovy

```diff
@@ -57,6 +81,15 @@ class OccurrenceController {
         def start = System.currentTimeMillis()
         requestParams.fq = params.list("fq") as String[] // override Grails binding which splits on internal commas in value
 
+        def (categoryToFqMap, fqToCategoryMap) = getDefaultFilters()
+
+        // if first time visit, apply default filters
```

Comment:
What is the design for this when cookies are cleared and the previous URL is shared with someone? In its current form it looks like it may try to add them again because `categoryToFqMap` has `List` as its value instead of `Set` to deduplicate the `fq` parameters.